CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v1.5.0 - 2023-01-13
+
+### Added
+
+- #112 - Added tests.
```

Comment:
There was also the option to create a CloudVision device and create RelationshipAssociations to the managed devices in CVP which was #121. Also to track the version for CloudVision in DLC.

Reply:
Updated.